src/components/AttributeSection.vue

```diff
@@ -1,30 +1,33 @@
-<!--
-  Displays entire section of attributes
- receives events from child component and check if change is possible in resources
- if changes are possible, emit event to top component to make changes, otherwise don't
- -->
-
-<script>
+<script setup>
+/**
+ * Displays entire section of attributes
+ * receives events from child component and check if change is possible in resources
+ * if changes are possible, emit event to top component to make changes, otherwise don't
+ */
 import { statSectionMixin } from "./mixins/statSectionMixin";
 import { attributesAndSkillsMixin } from "./mixins/attributesAndSkillsMixin";
 import Stat from "./Stat.vue";
 import RestrictionState from "./RestrictionState.vue";
 
+const emit = defineEmits(['statSectionHover'])
+
+</script>
```

Comment:
nepaci sa mi ze to nekonci stejnym tagom, ze tam neni specificky setup v end tagu, ale s tym sa asi neda nic robit